src/renderer/css/components/ExtraBar.css

```diff
@@ -13,6 +13,10 @@
     display: none;
 }
 
+#extra-bar.dark-mode {
+    background-color: #000;
```

Comment:
I'm not clear on what we're solving here. Is there a Jira ticket to reference with more detail on the isseu?

Reply:
@devinbinnie I updated css to use colors from TabBar.css and signed CLA

Here is an example of how it would look
![image](https://user-images.githubusercontent.com/45757556/228747552-55cd6fb2-3c7d-4468-b834-56dcb2a12e8d.png)

@matthewbirtch On windows and mac when using dark mode extra bar looks like this, it can also be hidden and then empty white space remains
![image](https://user-images.githubusercontent.com/45757556/228748264-95d0fafe-8ef4-4dfe-8d38-954c0210bad5.png)

Reply:
@matthewbirtch No JIRA ticket, but the above screenshot is what is being referred to. I think we should change the bar to be a similar colour to the header, but not the same colour. I'd still recommend the one I mentioned above, unless Matt has a different opinion.

Reply:
@asaadmahmood can I ask you to have a look at this one and provide a recommendation? @devinbinnie what are the cases where this `< Back` button and bar show?

Reply:
@devinbinnie @zoltan-ofir We can use `#1F1F1F`

Reply:
@zoltan-ofir Would you be able to update the colour for this?